### DISCUSSION TITLE
How did copilot autocomplete my api keys?

### DISCUSSION THREAD

**Discussion by attiasa22** [aws/copilot-cli#3618] [General]
I received my project's API keys through iMessage from my partner. When I created our config file for the .gitignore file, GitHub copilot autocompleted the key variables with the codes from my texts. 

Does GitHub copilot have permission to do this?

**Comment by Lou1415926** [comment]
Hello, this is AWS Copilot, not GitHub Copilot.
Looks like [this](https://github.com/github/feedback/discussions/categories/copilot-feedback) is the place to post your issue for them.
Good luck!
